src/ck-core/ckrdma.h

```diff
@@ -413,11 +410,6 @@ int getRootNode(envelope *env);
 // Function declaration for EM Ncpy Ack handler initialization
 void initEMNcpyAckHandler(void);
 
-struct zcPupIncompleteInfo {
```

Comment:
This type was unused

Reply:
Yeah, looks like this was added during the ZC Pup patch, but was never used.